src/main/java/bowling/controller/BowlingController.java

```diff
@@ -24,35 +25,54 @@ public BowlingController(InputView inputView, OutputView outputView) {
     }
 
     public void run() {
-        final Player player = new Player(playerName());
-        outputView.printScoreBoard(player);
+        final Players players = players();
+        outputView.printScoreBoard(players);
 
+        playGame(players);
+    }
+
+    private Players players() {
+        final int playerCount = inputView.inputPlayerCount();
+        final Players players = Players.create();
+        for (int i = 1; i <= playerCount; i++) {
+            players.add(Player.from(playerName(i)));
+        }
+        return players;
+    }
+
+    private void playGame(Players players) {
         for (int i = RoundNumber.MIN; i <= RoundNumber.MAX; i++) {
-            bowl(player, i);
+            bowlAll(players, i);
         }
     }
 
-    private void bowl(Player player, int i) {
+    private void bowlAll(Players players, int i) {
```

Comment:
```suggestion
    private void bowlAll(Players players, int ???) {
```
for 문을 벗어났으니 `i` 보다는 의미있는 이름을 지어주는게 좋겠습니다.

Reply:
그렇네요. 바꿔야겠습니다.